src/sdk/person/usePersonQuery.ts

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from 'react'
+import { gql } from '@vtex/graphql-utils'
+import type {
+  PersonQueryQuery,
+  PersonQueryQueryVariables,
+} from '@generated/graphql'
+import { useSession } from '@faststore/sdk'
+
+import { useQuery } from '../graphql/useQuery'
+import type { QueryOptions } from '../graphql/useQuery'
+
+export const query = gql`
+  query PersonQuery {
+    person {
+      id
+      email
+      givenName
+      familyName
+      isAuthenticated
+    }
+  }
+`
```

Comment:
Put it at the end of the file, before the export default, I think this is the pattern

Reply:
I don't think this is standard in this repo @igorbrasileiro. The other SDK hooks that include GraphQL queries are also defining them first